docs/REST API/rest-api.md

```diff
@@ -8822,6 +8822,7 @@ Get a list of all software.
 | available_for_install   | boolean | query | If `true` or `1`, only list software that is available for install (added by the user). Default is `false`.                                                                |
 | self_service            | boolean | query | If `true` or `1`, only lists self-service software. Default is `false`.  |
 | packages_only           | boolean | query | If `true` or `1`, only lists packages available for install (without App Store apps).  |
+| platforms               | string | query |  Platform to filter software titles by. Defaults to all platforms.  |
```

Comment:
```suggestion
| platforms               | string | query |  Comma-separated list of platforms to filter software titles by. (Options are: `"macos"`, `"windows"`, `"linux"`, `"chome"`, `"iOS"`, `"iPadOS"`.) If empty, includes all platforms.  |
```

Reply:
@rachaelshaw I think if we want to go with approach that Vicror proposed in a comment above, then this description needs to be adjusted.